doc/tutorial/code/exercises/Ex12.SHA1.fst

```diff
@@ -1,23 +1,23 @@
 module Ex12.SHA1
 open FStar.All
 open FStar.Seq
-open Platform.Bytes
+open FStar.Bytes
 open CoreCrypto
 
-type text  = bytes    (* a type abbreviation, for clarity *)
+type text  = b:bytes{UInt.fits (length b) 31}    (* a type abbreviation, for clarity *)
```

Comment:
So that two of them fit into `... 32`. The new bytes need size checks everywhere, but I don't know where exactly we should put them. (See also all the assumptions later.)